### DISCUSSION TITLE
FadCam v1.2.0

### DISCUSSION THREAD

**Discussion by anonfaded** [anonfaded/FadCam#60] [Announcements]
# What's New? Everything! 🎉  

### New Features:  
- 🌐 **Added translations**: Support for Chinese, Arabic, French, Turkish, Indonesian, and Pashto.  
- 🎥 **Framerate selection options**: Choose your desired framerate! *(Note: Framerates above 60 fps require root access. Tested on a Pixel device.)*  
- 🔒 **Wake lock and battery optimization disabled**: Prevent CPU sleep during recording for uninterrupted performance.  
- ⏸️ **Pause and resume recording**: Automatically pauses video and provides a resume notification if a camera error occurs.  
- 📲 **Notification enhancements**:  
  - Added a button to stop video recording.  
  - Tap the notification to open the app directly.  

### Improvements:  
- 🛠️ **Widget sizes fixed** for a better user experience.  
- 🔊 **Audio quality improved** for clearer recordings.  
- 📰 **Notifications fixed**: Fully functional on Android 12 and above.  
- 📦 **Enhanced video compression**: Now reduces space usage by up to 50%.  
- 🛠️ **App crash resolved**: Stable on Android 14 and 15.  

---

## 👥 Contributors:  

- **[@s0llvan](https://github.com/s0llvan)**: Contributions in [#23](https://github.com/anonfaded/FadCam/issues/23), [#36](https://github.com/anonfaded/FadCam/pull/36), [#29](https://github.com/anonfaded/FadCam/issues/29), and [#43](https://github.com/anonfaded/FadCam/issues/43).  
- **[@TheImperialOne](https://github.com/TheImperialOne)**: Contribution in [#25](https://github.com/anonfaded/FadCam/issues/25).  
- **[@91208347](https://github.com/91208347)**: Contribution in [#52](https://github.com/anonfaded/FadCam/pull/52).  
- **[@sonalder-darlene](https://github.com/sonalder-darlene)**: Contribution in [#45](https://github.com/anonfaded/FadCam/pull/45).  
- **[@mikropsoft](https://github.com/mikropsoft)**: Contribution in [#33](https://github.com/anonfaded/FadCam/pull/33).  
- **[@WeiguangTWK](https://github.com/WeiguangTWK)**: Contribution in [#27](https://github.com/anonfaded/FadCam/pull/27).  

---

# APK Files  

| APK File                              | Description                                    |
|---------------------------------------|------------------------------------------------|
| FadCam-v1.2.0-universal.apk           | Compatible with all devices (common)          |
| FadCam-v1.2.0-arm64-v8a.apk           | For 64-bit ARM devices (common)               |
| FadCam-v1.2.0-armeabi-v7a.apk         | For 32-bit ARM devices (less common)          |
| FadCam-v1.2.0-x86-64.apk              | For 64-bit Intel devices (rare)               |
| FadCam-v1.2.0-x86.apk                 | For 32-bit Intel devices (rare)               |

> [!TIP]
> 📝 **Note**: If you're unsure, download the `arm64` APK. If you encounter issues, use the universal APK instead.

---

Thank you for your continued support! 🚀

<hr /><em>This discussion was created from the release <a href='https://github.com/anonfaded/FadCam/releases/tag/v1.2.0'>FadCam v1.2.0</a>.</em>

**Comment by Vipintomar0** [comment]
Hi Team,
I would like to work with you

**Reply by anonfaded** [reply]
Sure. Open source contributions are welcomed. Anyone can contribute.
You can join Discord group as well where we discuss about issues, or any progress, or chat with users and open source contributors.
